model/src/wav_comp_nuopc.F90

```diff
@@ -574,10 +576,24 @@ subroutine InitializeRealize(gcomp, importState, exportState, clock, rc)
       write(stdout,'(a)')'--------------------------------------------------'
     end if
 
+    call ESMF_ClockPrint(clock, options="startTime", preString="Model Start Time: ", &
+         unit=msgString, rc=rc)
+    call ESMF_LogWrite(trim(msgString), ESMF_LOGMSG_INFO)
+    call ESMF_ClockPrint(clock, options="currTime", preString="Model Current Time: ", &
+         unit=msgString, rc=rc)
+    call ESMF_LogWrite(trim(msgString), ESMF_LOGMSG_INFO)
+    call ESMF_ClockGet( clock, startTime=startTime, currTime=currTime, rc=rc)
+    TimeOffset = currTime - startTime
+    call ESMF_TimeIntervalGet(TimeOffset, h_r8=toff, rc=rc)
+    write(msgstring,'(a,g14.7)')'TimeOffset: CurrTime - StartTime = ',toff
+    call ESMF_LogWrite(trim(msgString), ESMF_LOGMSG_INFO)
     ! Initial run or restart run
     if ( runtype == "initial") then
       call ESMF_ClockGet( clock, startTime=esmfTime, rc=rc)
       if (ChkErr(rc,__LINE__,u_FILE_u)) return
+#ifndef CESMCOUPLED
```

Comment:
This should be corrected as `#ifndef W3_CESMCOUPLED`. Other than this, it all looks good to me.

Reply:
Good catch.